ui/app/templates/vault/cluster/replication-dr-promote/details.hbs

```diff
@@ -32,7 +32,7 @@
       {{!-- passing in component to render so that the yielded components are flexible based on the dashboard --}}
         @componentToRender='replication-secondary-card' as |Dashboard|>
         <Dashboard.card
-          @title="States"
+          @title="Status"
```

Comment:
this fixed a bug in which the secondary dashboard rendered 2 cards with the `known_primary_cluster_addr` content inside it